test/sleep/main.c

```diff
@@ -128,6 +221,67 @@ static int test_nanosleep_10ms() {
     return check_nanosleep(&period_of_10ms);
 }
 
+static int test_clock_nanosleep_for_interval_time() {
+    struct timespec period;
+
+    // CLOCK_REALTIME with 0s
+    period.tv_sec = 0;
+    period.tv_nsec = 0;
+    if (check_clock_nanosleep_interval_with_clockid(CLOCK_REALTIME, &period) != SUCCESS) {
+        printf("check_clock_nanosleep_interval failed with peroid={ %lds , %ld ns }, clock_id=%d\n",
```

Comment:
you can just use THROW_ERROR here, which will print and return -1.

Reply:
`THROW_ERROR()` will show the `errno` (See https://github.com/occlum/ngo/blob/master/test/include/test.h#L25-L29). The `errno` will remain unchanged if a successful syscall returns. The `errno`(!=0) in `THROW_ERROR()` may be confused, so I use `printf()` here.